components/automate-cli/pkg/verifyserver/utils/userutils/user.go

```diff
@@ -0,0 +1,27 @@
+package userutils
+
+import "os/user"
+
+type UserUtil interface {
+	Lookup(name string) (*user.User, error)
+	LookupGroup(name string) (*user.Group, error)
+	LookupGroupId(name string) (*user.Group, error)
+}
+
+type UserUtilImp struct{}
+
+func NewUserUtilImp() UserUtil {
+	return &UserUtilImp{}
+}
+
+func (s *UserUtilImp) Lookup(name string) (*user.User, error) {
+	return user.Lookup(name)
+}
+
+func (s *UserUtilImp) LookupGroup(name string) (*user.Group, error) {
+	return user.LookupGroup(name)
+}
+
+func (s *UserUtilImp) LookupGroupId(name string) (*user.Group, error) {
+	return user.LookupGroupId(name)
+}
```

Comment:
Add one more empty line at the end of this file

Reply:
Done